.github/workflows/need_reply_remove.yml

```
@@ -11,6 +11,8 @@ jobs:
     if: |
       github.event.comment.author_association != 'OWNER' &&
       github.event.comment.author_association != 'COLLABORATOR' &&
+      github.repository_owner == 'pybamm-team' && 
```

```suggestion
      github.repository_owner == 'pybamm-team' &&
```
This shuld be causing the style failure.

---

@santacodes, to run the style checks you can do `pip install pre-commit` and `pre-commit install` to install hooks in your `.git/` folder. This will stop commits from being pushed that do not comply with the style guidelines. Another way is to run `nox -s pre-commit`, which is a little expensive.